javascript/javascript3/week2/fetchandwait.js

```diff
@@ -0,0 +1,26 @@
+const astronautApi = "http://api.open-notify.org/astros.json";
+    const promise = new Promise((resolve, reject) => {
+      setTimeout(() => {
+      resolve("timeout after 3 seconds");
+      }, 3000);
+    });
+
+    promise.then(() => {
+      fetch(astronautApi)
+        .then((response) => {
+            response.json()
+        })
+        .then((data) => console.log(data));
+    });
+    //using async/await
+    async function astronaut() {
```

Comment:
_astronaut_ is a noun, a function name should be a _verb_. What _happens_ when you call this function?